Text Editor Debug Fail

`cd examples\C\text-editor`
`mingw32-make debug`

```
"Build Text Editor Example (debug)..."
.../../lib/libmingw32.a(lib64_libmingw32_a-crtexewin.o):crtexewin.c:(.text.startup+0xbd): undefined reference to `WinMain'

```

--- a/examples/C/text-editor/Makefile
+++ b/examples/C/text-editor/Makefile
@@ -47,7 +47,7 @@ all: release
 
 debug: --setup
 	@echo "Build Text Editor Example ($@)..."
-	@$(CC) -g -Wall $(LDFLAGS) -o $(OUT)
+	@$(CC) -g -Wall $(CFLAGS) $(LDFLAGS) -o $(OUT)
 	@echo "Done."
 
 release: --setup

